bucket/caffeine.json

```
@@ -0,0 +1,15 @@
+{
+    "homepage": "https://www.zhornsoftware.co.uk/caffeine/index.htm",
+    "description": "Prevent PC standby or sleep",
+    "version": "1.64",
+    "license": "Unknown",
+    "url": "https://www.zhornsoftware.co.uk/caffeine/caffeine.zip#/dl.7z",
+    "hash": "5cbaaefaf4c24d3af73b856c98b079ff4bcdf01ca104b179733fca0616a05d8d",
+    "bin": "caffeine.exe",
+    "shortcuts": [
+        [
+            "caffeine.exe",
+            "Caffeine"
+        ]
+    ]
```

Since versions were updated even after year there is no reason to not include autoupdate

```suggestion
    ],
    "checkver": "v([\\d.]+)\\s+-",
    "autoupdate": {
        "url": "https://www.zhornsoftware.co.uk/caffeine/caffeine.zip"
    }
```